python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-name-server-groups.j2

```diff
@@ -0,0 +1,27 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{% for name_server_group in ip_name_server_groups | arista.avd.natural_sort("name") %}
+!
+ip name-server group {{ name_server_group.name }}
+{%     for name_server in name_server_group.name_servers | arista.avd.natural_sort("ip_address") %}
+{%         if name_server.priority is not arista.avd.defined %}
+{%             set name_server_cli = "name-server vrf " ~ name_server.vrf | arista.avd.default("default") ~ " " ~ name_server.ip_address %}
+   {{ name_server_cli }}
+{%         endif %}
+{%     endfor %}
+{%     for name_server in name_server_group.name_servers | arista.avd.natural_sort("ip_address") %}
```

Comment:
for priority the sequence is base on priority then for same priority it is base on ip address. In case of VRF, the sequence as follow priority->vrf->ip address

Reply:
```
s1-leaf1(config-ns-mynameserver1)#sh ac
ip name-server group mynameserver1
   name-server vrf default 10.0.0.6
   name-server vrf CCC 10.0.0.8 priority 1
   name-server vrf CCC 12.0.0.8 priority 1
   name-server vrf DDDD 10.0.1.10 priority 1
   name-server vrf MGMT 10.0.1.10 priority 1
   name-server vrf default 10.0.0.10 priority 1
   name-server vrf default 10.1.1.1 priority 1
   name-server vrf default 12.1.1.1 priority 1
   name-server vrf default 13.1.1.1 priority 1
   name-server vrf default 11.1.1.1 priority 2
   name-server vrf default 1.1.1.1 priority 3
   name-server vrf default 10.0.0.5 priority 4
```

Reply:
Fixed.